src/main/resources/templates/script-plugin/filecopier/filecopier.template

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+
```

Comment:
Run this template (and any other bash scripts) through shellcheck to be sure conventions like `set -eu` are used.

Reply:
ok working in that